spec/01-unit/08-router_spec.lua

```diff
@@ -4251,3 +4251,96 @@ describe("[both regex and prefix with regex_priority]", function()
 
 end)
 
+
+describe("incremental rebuild", function()
```

Comment:
Could we move these code to line 1673, which tests `flavor ~= "traditional"`.

Reply:
@dndx , Could you take a look at these cases, Do you have better idea?

Reply:
Moved.